docs/content/latest/explore/multi-region-deployments/follower-reads-ysql.md

```diff
@@ -9,25 +9,245 @@ menu:
   latest:
     identifier: explore-multi-region-deployments-follower-reads-ysql
     parent: explore-multi-region-deployments
-    weight: 285
+    weight: 280
 isTocNested: true
 showAsideToc: true
 ---
 
 <ul class="nav nav-tabs-alt nav-tabs-yb">
-  
   <li >
     <a href="../follower-reads-ysql/" class="nav-link active">
       <i class="icon-postgres" aria-hidden="true"></i>YSQL</a>
   </li>
-  
+
   <li >
     <a href="../read-replicas-ycql/" class="nav-link">
       <i class="icon-cassandra" aria-hidden="true"></i>YCQL</a>
-  </li>  
-
+  </li>
 </ul>
 
-YSQL follower reads support is currently in Beta. 
+## Introduction
+
+YugabyteDB requires reading from the **leader** to read the latest data. However, for applications that do not require the latest data, and/or are working with unchanging data; the cost of contacting a potentially remote leader to fetch the data may be wasteful. The application could benefit with better latency by reading from a **replica** that may be closer to the client.
+
+Since the replicas may not be up to date with respect to all the updates, it should be noted that this design may respond with stale data. The user may specify how much staleness the application can tolerate. When enabled, read-only operations may be handled by the closest replica; instead of having to go to the leader.
+
+## Surface area and Usage
+
+### Surface area
+
+- *yb_read_from_followers* : This session variable controls whether read from followers is enabled.
+
+- *yb_follower_read_staleness_ms* : This session variable defines the maximum staleness that the user is willing to tolerate.
+
+### Default behavior
+
+- *yb_read_from_followers* : Defaults to false.
+
+- *yb_follower_read_staleness_ms* : Defaults to 30 seconds.
+
+### Expected behavior
+
+The table describes what the expected behavior is when a read happens from a follower.
+
+| Conditions | Expected behavior |
+| --- | --- |
+| yb_read_from_followers is true AND <br> (Transaction is marked read-only <br>OR<br> Single-statement-select can be inferred to be read-only) <br> | Read will happen from the closest follower|
+|yb_read_from_followers is false OR <br> Transaction/statement is not read only | Read will happen from the leader |
+
+### Read from follower conditions
+
+- If the follower’s safe-time is larger than the read-time thus chosen, the follower may serve the read without any delay.
+
+- If the follower is not yet caught up to the specified timestamp, the read will be redirected to a different replica transparently from the end-user. The end user may see a slight increase in latency depending on the location of the replica which satisfies the read.
```

Comment:
"the specified timestamp" -- which timestamp? Again, this concept has not been explained.

Reply:
If the follower is not yet caught up to `<current_time> - <staleness> `...